source/Irrlicht/COpenGLDriver.h

```diff
@@ -111,13 +111,13 @@ namespace video
             GLboolean stencilTestEnable = 0;
             GLboolean multisampleEnable = 1;
             struct SClipCtrl {
-                GLenum origin;
-                GLenum depth;
+                GLenum origin = GL_LOWER_LEFT;
+                GLenum depth = GL_NEGATIVE_ONE_TO_ONE;
                 bool operator!=(const SClipCtrl& rhs) const { return origin!=rhs.origin || depth!=rhs.depth; }
             } clipControl;
             struct SDepthRng {
-                GLclampd znear;
-                GLclampd zfar;
+                GLclampd znear = 0.0;

```

Comment:
our engine default is reverse Z

Reply:
resolved (it's GL default, not our default)